### ISSUE TITLE
QR Code Scanner not properly detecting QR Codes

### ISSUE THREAD

**Issue by mightyBroccoli** [siacs/Conversations#2866]
#### General information
Today I tried to verify a friend via the included QR Code Scanner, on his phone and it did not work at all. I tried my normally installed QR Code scanner, which worked instantly. As Conversation does not work with a 3 party any more it was not possible for me to verify the code.

* **Version:** 2 beta 2
* **Device:** Samsung S7 stock
* **Android Version:** 7
* **Server name:** magicbroccoli.de
* **Server software:** prosody-0.10
* **Installed server modules:** completely green on [Compliance](https://conversations.im/compliance/)
* **Conversations source:** PlayStore Beta Channel

#### Steps to reproduce

1. get a friend.
2. convince him/her to use XMPP and Conv
3. enable Omemo and try to verify with the integrated scanner.

#### Expected result

1. everlasting friendship

#### Actual result

Does not detect at all.

Should I try to get a log from this? I think this should be really easy to replicate by oneself.

**Comment by iNPUTmice** [comment]
Which of the scan buttons did you use?

I don't understand what does not detect at all mean? Does the scanner not recognize the QR code? (Meaning the scanner remains open for ever?)

**Comment by mightyBroccoli** [comment]
Hmm I connected the logcat and captured the log.But then it worked.

I think the distance the scanner needs to be held in front of the qr code. I found that though pure luck as my cable was not long enough and suddenly the scanner worked 🦄 
I tried to capture some logs from the case that the scanner was not working but there were no loglines regarding this or for a fact any log lines. 

I used two methods, first the button below a friends keys to open the scanner and then directly selecting a key holding it and selecting verify via barcode.

In the end both work when the distance between my phone and the qr code is about 20 cm to about 30 cm.
